tools/topology/topology2/cavs-es83x6.conf

```diff
@@ -0,0 +1,254 @@
+	DMIC0_DAI_GAIN			'eqiir.14.1'
+	DMIC0_NAME			'dmic01'
+	DMIC0_PCM_CAPS			'Gain Capture 13'
+	# override BE link ids
+	NUM_HDMIS			3
+	HDMI1_ID			3
+	HDMI2_ID			4
+	HDMI3_ID			5
+	PLATFORM 			"none"
+	USE_CHAIN_DMA			"false"
+	DEEP_BUFFER_PIPELINE_ID		15
+	DEEP_BUFFER_PCM_ID		31
+	DEEP_BUFFER_PIPELINE_SRC	'mixin.15.1'
+	DEEP_BUFFER_PIPELINE_SINK	'mixout.2.1'
+	DEEP_BUFFER_PCM_NAME		'Deepbuffer Jack Out'
+	HEADSET_CODEC_NAME		'SSP1-Codec'
+	HEADSET_PLAYBACK_PCM_STREAM_NAME 'Headset Playback'
+	HEADSET_CAPTURE_PCM_STREAM_NAME	'Headset Capture'
+	HEADSET_SSP_DAI_INDEX		1
+	HEADSET_HW_CONFIG_NAME		'HEADSET HWCFG'
+	HEADSET_PCM_NAME		"Headset"
+	HEADSET_PCM_ID			0
+}
+
+# override defaults with platform-specific config
+IncludeByKey.PLATFORM {
+	"mtl"	"platform/intel/mtl.conf"
+}
+
+# include DMIC config if needed.
+IncludeByKey.NUM_DMICS {
+	"[1-4]"	"platform/intel/dmic-generic.conf"
+}
+
+# include HDMI config if needed.
+IncludeByKey.NUM_HDMIS {
+	"[3-4]"	"platform/intel/hdmi-generic.conf"
+}
+
+# include deep buffer config if buffer size is in 1 - 1000 ms.
+IncludeByKey.DEEPBUFFER_FW_DMA_MS{
+	"([1-9]|[1-9][0-9]|[1-9][0-9][0-9]|1000)" "platform/intel/deep-buffer.conf"
+}
+
+#
+# List of all DAIs
+#
+#SSP Index: 0, Direction: duplex
+Object.Dai.SSP [
+	{
+		id 		0
+		dai_index	$HEADSET_SSP_DAI_INDEX
+		direction	"duplex"
+		name		$HEADSET_CODEC_NAME
+		default_hw_conf_id	0
+		sample_bits		32
+		io_clk		$MCLK
+
+		Object.Base.hw_config.1 {
+			name	$HEADSET_HW_CONFIG_NAME
+			id	0
+			bclk_freq	3072000
+			tdm_slot_width	32
+			# TODO: remove this. Needs alsaptlg change.
+			Object.Base.link_config.1 {
+				clock_source	1
+			}
+		}
+	}
+]
+
+#
+# Pipeline definitions
+#
+# PCM1 ---> gain ----> Mixin ----> Mixout ----> gain ----> SSP1
+#
+# SSP1 ----> PCM0
+
+# Pipeline ID:1 PCM ID: 0
+Object.Pipeline {
+	# playback pipelines
+	host-copier-gain-mixin-playback [
+		{
+			index	1
+
+			Object.Widget.host-copier.1 {
+				stream_name	'$HEADSET_PLAYBACK_PCM_STREAM_NAME'
+				pcm_id $HEADSET_PCM_ID
+			}
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name	'Playback Volume 1'
+				}
+			}
+		}
+	]
+
+	mixout-gain-dai-copier-playback [
+		{
+			index	2
+
+			Object.Widget.dai-copier.1 {
+				dai_index $HEADSET_SSP_DAI_INDEX
+				dai_type	"SSP"
+				copier_type	"SSP"
+				stream_name	"$HEADSET_CODEC_NAME"
+				node_type $I2S_LINK_OUTPUT_CLASS
+			}
+
+			Object.Widget.gain.1 {
+				Object.Control.mixer.1 {
+					name	'Main Playback Volume 2'
+				}
+			}
+		}
+	]
+
+	host-gateway-capture [
+		{
+			index	5
+
+			Object.Widget.host-copier.1 {
+				stream_name	'$HEADSET_CAPTURE_PCM_STREAM_NAME'
+				pcm_id $HEADSET_PCM_ID
+			}
+		}
+	]
+
+	io-gateway-capture [
+		{
+			index	6
+			direction	capture
+
+			Object.Widget.dai-copier."1" {
+				dai_index	$HEADSET_SSP_DAI_INDEX
+				dai_type	"SSP"
+				type		dai_out
+				copier_type	"SSP"
+				stream_name	"$HEADSET_CODEC_NAME"
+				node_type	$I2S_LINK_INPUT_CLASS
+				Object.Base.audio_format.1 {
```

Comment:
Please use `Object.Base.input_audio_format` and `Object.Base.output_audio_format`

Reply:
Updated, please check the latest